support-frontend/assets/pages/contributions-landing/contributionsLanding.scss

```diff
@@ -934,8 +936,17 @@ form {
 //***************************
 // Campaign styles:
 //***************************
+.gu-content--toxic-america {
```

Comment:
Toxic america - is this still running or will it be reused in future?

Reply:
good spot! This can go

Reply:
You're so right - we should generalise this, but right now this is where we are keeping all of our standard campaign template CSS and we update it whenever we have a new campaign. We can change this but let's not include as part of this PR.

Reply:
👍